label_sync/README.md

```diff
@@ -0,0 +1,47 @@
+# label_sync
+
+Update or migrate github labels on repos in a github org based on a YAML file
+
+## Configuration
+
+A typical labels.yaml file looks like:
+
+```yaml
+---
+labels:
+  - color: 00ff00
+    name: lgtm
+  - color: ff0000
+    name: priority/P0
+    previously:
+    - color: 0000ff
+      name: P0
+  - name: dead-label
+    color: cccccc
+    deleteAfter: 2017-01-01T13:00:00Z
+```
+
+This will ensure that:
+
+- there is a green `lgtm` label
+- there is a red `priority/P0` label, and previous labels should be migrated to it:
+  - if a `P0` label exists:
+    - if `priority/P0` does not, modify the existing `P0` label
+    - if `priority/P0` exists, `P0` labels will be deleted, `priority/P0` labels will be added
+- if there is a `dead-label` label, it will be deleted after 2017-01-01T13:00:00Z
```

Comment:
<!--mdfmt-bot-->
Format markdown:
```suggestion
    - if `priority/P0` exists, `P0` labels will be deleted, `priority/P0` labels
      will be added
- if there is a `dead-label` label, it will be deleted after
  2017-01-01T13:00:00Z
```